src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-28 calendar--start-day-wed">
```

Comment:
- Modifiers are not working. Fix it, please 
![image](https://github.com/mate-academy/layout_calendar/assets/99403198/88a81bab-d370-4af8-89ce-554ee4f6d222)